index.js

```diff
@@ -281,12 +281,20 @@ function decorateFunctions(node) {
     // Pair up the function params with the JSDoc params (if they exist)
     funcNode.node.params.forEach(function(param) {
         for (i = 0; i < funcNode.jsdoc.params.length; i++) {
-            if (funcNode.jsdoc.params[i].name === param.name &&
-                    funcNode.jsdoc.params[i].type) {
-                // replace the function param name with the type annotated param
-                param.update(
-                    param.source() + ": " + funcNode.jsdoc.params[i].type
-                );
+            if (funcNode.jsdoc.params[i].type) {
+                if (param.type === 'AssignmentPattern') {
+                    if (funcNode.jsdoc.params[i].name === param.left.name) {
```

Comment:
Indeed probably good enough to just follow jsdoc. Flow itself will fail anyway because you assign a string to a number which indicates the jsdoc is wrong.